src/binaryen-c.cpp

```diff
@@ -1030,4 +1072,15 @@ void BinaryenSetAPITracing(int on) {
   }
 }
 
+//
+// ===== Emscripten support =====
+//
+
+#ifdef EMSCRIPTEN
+int __cxa_thread_atexit(void(*)(char*), char*, char*)
```

Comment:
Oh, except to remove the definitions of `__cxa_thread_atexit`. Which I guess we can only do once that other PR lands.